tools/webpack/script-modules.js

```diff
@@ -102,7 +104,13 @@ module.exports = {
 	resolve: {
 		extensions: [ '.js', '.ts', '.tsx' ],
 	},
-	plugins: [ ...plugins, new DependencyExtractionWebpackPlugin() ],
+	plugins: [
+		...plugins,
+		new DependencyExtractionWebpackPlugin( {
+			combineAssets: true,
+			combinedOutputFile: `./assets.php`,
```

Comment:
Huge perf win! It will pay off over time. 👏

Reply:
I was trying to remember why there are two versions in WP core:

https://github.com/WordPress/wordpress-develop/blob/35907da95151b45d197102ea4d69e10ad019cab9/tools/webpack/packages.js#L201

All good here. I figured out that in WP core, there are two parallel build processes - one for `development` and one for `production,` so they shouldn't be overridden. It isn't the case here.

Reply:
On the second thought:

`npm run dev`

<img width="1299" alt="Screenshot 2024-09-20 at 12 14 31" src="https://github.com/user-attachments/assets/2b9a0f6e-51cb-45ce-912c-d7a7b2ad3497">

`npm run build`

<img width="1414" alt="Screenshot 2024-09-20 at 12 19 54" src="https://github.com/user-attachments/assets/db373c1a-6695-49de-abd7-873a37e187ff">

Notice the file name difference `.js` vs `.min.js`.